tests/inst/.gitignore

```
@@ -0,0 +1,2 @@
+target/
```

```suggestion
/target
```

https://github.com/rust-lang/cargo/pull/4944